src/driver/aurora-data-api-pg/AuroraDataApiPostgresDriver.ts

```diff
@@ -90,7 +90,19 @@ export class AuroraDataApiPostgresDriver extends PostgresWrapper implements Driv
      * Creates a query runner used to execute database queries.
      */
     createQueryRunner(mode: ReplicationMode) {
-        return new AuroraDataApiPostgresQueryRunner(this, mode);
+        return new AuroraDataApiPostgresQueryRunner(
+            this,
+            new this.DataApiDriver(
```

Comment:
Do we need the client on the driver still if we have it here?

Reply:
Yeah it runs a couple of statements like enabling postgres extensions.

Reply:
The base `PostgresDriver` effectively uses a query runner to accomplish this - by calling `this.connection.query` which creates a new query runner & executes them.

Can we rely on the base `PostgresDriver` functionality instead of keeping an extra connection open at all times for this?

Reply:
Good catch, will fix.